pkg/io/binaryReader.go

```diff
@@ -8,9 +8,9 @@ import (
 	"reflect"
 )
 
-// maxArraySize is a maximums size of an array which can be decoded.
+// MaxArraySize is a maximums size of an array which can be decoded.
```

Comment:
It'd be nice to fix some wording along the way here, `is the maximum`.

Reply:
FIxed.